src/modules/rlm_rest/rlm_rest.c

```diff
@@ -1192,6 +1201,12 @@ static int instantiate(module_inst_ctx_t const *mctx)
 		return -1;
 	}
 
+	cs = cf_section_find(conf, "connections", NULL);
+	if (!cs) cs = cf_section_alloc(conf, conf, "connections", NULL);
+
+	if (cf_section_rules_push(cs, conn_config) < 0) return -1;
+	if (cf_section_parse(inst, &inst->conn_config, cs) < 0) return -1;
```

Comment:
This shouldn't be needed, if it is, that's a bug in the cf_parse code... Or it looks like you might just be missing the FR_TYPE_SUBSECTION entry in `module_config`?